census/models.py

```diff
@@ -23,9 +23,9 @@ def first_date(self):
 
     title = models.CharField(max_length=100, help_text="Title or short description of the event")
     description = models.TextField(blank=True, help_text="Full description of the event")
-    recurrences  = RecurrenceField(default=None, blank=True, verbose_name='Date Information', help_text="Add date of the event or rules for a recurring event")
-    start_time = models.TimeField(help_text="What time does this event start? HH:MM:AM/PM")
-    end_time = models.TimeField(help_text="What time does the event end? HH:MM:AM/PM")
+    recurrences = RecurrenceField(default=None, blank=True, help_text="This event occurs more than once.")
+    start_datetime = models.DateTimeField(help_text="When does the event start?")
```

Comment:
We don't seem to get a calendar widget on the datetime fields.  I thought we had this before.

Reply:
Yes, when we were using Django's built in widgets, but now we are writing custom HTML ourselves for uswds and we'll have to do some work to get the Widget working. I won't be able to do that as part of this PR.